### PR TITLE
avicii: Initial v10.1 release

### PR DESCRIPTION
**Pull request by psavarmattas**
Adding support for v10.x and pushing v10.1

### CODE REVIEW

File: crDroidAndroid-14.0-20240113-avicii-v10.1.json
Comment (gwolf2u) [inline crDroidAndroid-14.0-20240113-avicii-v10.1.json:1]: wrong file name

File: crDroidAndroid-14.0-20240113-avicii-v10.1.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+			"buildtype": "Monthly",
+			"forum": "https://xdaforums.com/t/rom-official-14-oneplus-nord-avicii-crdroid-v10-x.4650600/",
+			"gapps": "",
+			"firmware": "",
+			"modem": "",
+			"bootloader": "",
+			"recovery": "",
+			"paypal": "https://www.paypal.me/psmattas610",
+			"telegram": "https://t.me/psmbuilds",
+			"dt": "https://github.com/crdroidandroid/android_device_oneplus_avicii",
```
Comment (gwolf2u) [inline crDroidAndroid-14.0-20240113-avicii-v10.1.json]: permissive is not valid
https://github.com/crdroidandroid/android_device_oneplus_avicii/blob/14.0/BoardConfig.mk#L123

**Review by gwolf2u — CHANGES_REQUESTED** [review]

File: crDroidAndroid-14.0-20240113-avicii-v10.1.json (hunk 1 above)
Comment (psavarmattas) [reply crDroidAndroid-14.0-20240113-avicii-v10.1.json]: Sir the build is not permissive just forgot to push the latest changes to the DT. 

the commit is now made: https://github.com/crdroidandroid/android_device_oneplus_avicii/commit/09e555742e6064e70e61d0fc4aa3fcbba0c620bc

File: crDroidAndroid-14.0-20240113-avicii-v10.1.json
Comment (psavarmattas) [reply crDroidAndroid-14.0-20240113-avicii-v10.1.json:1]: Updated that!

**Review by gwolf2u — APPROVED** [review]

**Pull request merged**